src/firefly/java/edu/caltech/ipac/firefly/resources/master-table.csv

```diff
@@ -0,0 +1,51 @@
+name,missionLabel (project),missionId,dataProductLabel,dataProductId,acronym,instrumentId,instrumentLabel,wavebandId,wavebandDesc,wavelength (in microns),wavelengthDesc,projectTypeKEy,projectTypeDesc,minSizeDeg,maxSizeDeg,helpUrl,tooltip,apiType,atlasTable,dataType
+JSON DM-12001,project,,subProject,surveyKey,,,,surveyKeyBand,title,,,,,minRangeDeg,maxRangeDeg,,,Service,surveyKeyAlt,
+string,string,string,string,string,string,string,,string,string,float,string,,,float,float,,string,string,string,string
```

Comment:
i don't think line 2 and 3 should be in the CSV.  There should only be a title line.